src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,162 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.Scanner;
+
+/**
+ * Ui deals with interactions with the user.
+ */
+public class Ui {
+    private final static String LINE = "____________________________________________________________";
+    private final static String INDENTATION = "   ";
+    private Scanner sc;
+
+    /**
+     * Constructor for Ui.
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Reads user input.
+     * @return user input as string.
+     */
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Prints a line.
+     */
+    public void showLine() {
+        System.out.println(INDENTATION + LINE);
+    }
+
+    /**
+     * Prints welcome message.
+     */
+    public void showWelcome() {
+        System.out.println(INDENTATION + LINE);
```

Comment:
Similar to a previous comment, if `INDENTATION` is used every time you print to sys out, you could consider defining your own `print` method that takes in a String and adds the `INDENTATION` before printing.